Ghidra/Features/Base/src/main/java/ghidra/util/GhidraJarBuilder.java

```diff
@@ -338,11 +338,12 @@ public void buildSrcZip(File outputFile, TaskMonitor monitor)
 				wroteToZip |= writeZipRecursively(zip, srcDir.getAbsolutePath(), srcDir);
 			}
 		}
-		if (wroteToZip) {
-			System.out
-					.println("Can't create source zip!  Has source been downloaded and installed?");
+		if (!wroteToZip) {
+			System.out.println("Can't create source zip!  Has source been downloaded and installed?");
 			// zip.close reports error if nothing has been written to it
 			zip.close();
+		} else {
```

Comment:
pull zip.close out of `if` instead of two calls?

Reply:
Absolutely, I just wanted to highlight the fact that there previously wasn't a `close` call in the other case.